config/secrets.yml

```diff
@@ -1,8 +1,15 @@
-shared:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
 
 development:
+  secret_key_base: 35bba17b71c4f6ee1a4e45ab6c3917362a67b8478e45bfcaac728b7cfa717219ce0b7c3aeb67d4d98cb844c0ff29a06154b4483efa13a7cda1365fab5dad5415
+  facebook_app_id: <%= ENV["FACEBOOK_APP_ID"] %>
+  facebook_app_secret: <%= ENV["FACEBOOK_APP_SECRET"] %>
 
 test:
+  secret_key_base: 35bba17b71c4f6ee1a4e45ab6c3917362a67b8478e45bfcaac728b7cfa717219ce0b7c3aeb67d4d98cb844c0ff29a06154b4483efa13a7cda1365fab5dad5415
```

Comment:
You should move these keys to `.env`.

Look into using different environments with the dotenv-rails gem.